### PR TITLE
plugins: rts54hub: update rts54hub.quirk

### PR DESCRIPTION
**Pull request by AnyProblem**
add Acer Dock ID

Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [*] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/rts54hub/rts54hub.quirk [hunk 1]
```diff
@@ -38,3 +38,24 @@ FirmwareSizeMax = 0x90000
 Rts54TargetAddr = 0x20
 Rts54I2cSpeed = 0x2
 Rts54RegisterAddrLen = 0x04
+
+# Acer D501 Dock
+[USB\VID_2BEF&PID_1009]
+Plugin = rts54hub
+GType = FuRts54HubDevice
+Vendor = Realtek
```
Comment (hughsie) [inline plugins/rts54hub/rts54hub.quirk:46]: Not Acer?

Comment (AnyProblem) [reply plugins/rts54hub/rts54hub.quirk:46]: This VID is QSI( Quanta Storage Inc) which is ODM of Acer Dock
Acer do not customized ID on this project

Comment (hughsie) [reply plugins/rts54hub/rts54hub.quirk:46]: Okay, so why do we need the Realtek override? Usually we just set the vendor automatically based on the VID.

Comment (AnyProblem) [reply plugins/rts54hub/rts54hub.quirk:46]: okay, so just don't set the Vendor default is automatic?

Comment (hughsie) [reply plugins/rts54hub/rts54hub.quirk:46]: Yes, I think you can just remove the `Vendor=` line completely.

**Pull request merged**